src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementDialog.tsx

```diff
@@ -228,8 +271,32 @@ const PermissionManagementDialog = ({
                   userInheritedRoles={userInheritedRoles}
                   colonyDomains={colonyData.processedColony.domains}
                   onDomainSelected={setSelectedDomainId}
+                  userHasPermission={userHasPermission}
                 />
               </DialogSection>
+              {!userHasPermission && (
+                <DialogSection appearance={{ theme: 'sidePadding' }}>
+                  <div className={styles.noPermissionFromMessage}>
+                    <FormattedMessage
+                      {...MSG.noPermissionFrom}
+                      values={{
+                        firstRoleRequired: (
+                          <PermissionsLabel
+                            permission={ColonyRole.Funding}
+                            name={{ id: `role.${ColonyRole.Funding}` }}
+                          />
+                        ),
+                        secondRoleRequired: (
+                          <PermissionsLabel
+                            permission={ColonyRole.Administration}
+                            name={{ id: `role.${ColonyRole.Administration}` }}
+                          />
+                        ),
+                      }}
+                    />
+                  </div>
+                </DialogSection>
+              )}
```

Comment:
This requirement display is **wrong!**

Just above, in this file, you are checking for the `Architecture` permission:

https://github.com/JoinColony/colonyDapp/blob/6e965069641df996d29f3105799bf5c519f8dc50/src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementDialog.tsx#L212-L215

But when showing the user the error message, you are telling them it requires both `Funding` and `Adminstration` ?

![Screenshot from 2021-02-05 15-52-49](https://user-images.githubusercontent.com/1193222/107046499-5bbdf900-67cf-11eb-80d0-a7996e4ec015.png)

Please fix this